.gitignore

```diff
@@ -10,3 +10,4 @@ cpu.out
 .DS_Store
 .idea/
 vendor/
+.venv
```

Comment:
idk if this is generated by python vm compilation or some other script, but we don't use any python in this repo so try to remove it or compile cairo in some other dir / local repo

Reply:
Sorry, it was my own setting and I didn't mean to commit this, but now it's removed.